day-one/create-resource-from-template.mdx

```diff
@@ -1,6 +1,169 @@
 ---
 title: 'Create Resource from Live Template'
 sidebarTitle: 'Create from a Template'
-description: 'Empower enterprise developers to quickly create resources using fully configured golden paths with predefined settings.'
+description: "Quickly generate new services or blueprints based on pre-configured templates."
 icon: 'layer-group'
----
\ No newline at end of file
+---
+
+<Frame>
+    <img src="/images/day-one/resource-from-template.png" />
+</Frame>
+
+Amplication's **Templates** let you efficiently create new services or blueprints that align with organizational standards and accelerate your development.
+By using pre-configured templates, you can quickly generate new resources.
+
+## Types of Templates
+
+Amplication offers two distinct ways to create templates, giving you flexibility in how you standardize your resources:
+
+- **Templates from Services:** These templates capture the specific configurations and standards of an existing Amplication service. When you create a resource from a service-based template, you use Amplication's built-in opinionated approach, particularly when the original service uses the Node.js or .NET code generator.
+- **Templates from Blueprints:** Templates can also be created from Blueprints. This allows you to capture and reuse the broader organizational standards and configurations defined within a Blueprint for quicker instantiation of resources adhering to those guidelines.
```

Comment:
This should not be in this article..
From the developer perspective - they dont really care what is service or what is blueprint 
The idea to select a template to get all these selections behind the scene